src/handlers/transformation/build-event-name.ts

```diff
@@ -0,0 +1,34 @@
+import { CsvRow } from "./transform-row";
+
+interface Rules {
+  "Minimum Level Of Assurance": string;
+  "Billable Status": string;
+  "Event Name": string;
+}
+
+export interface EventNameRules {
+  [key: string]: Rules[];
+}
+
+export async function buildEventName(
```

Comment:
It's not really "building" the event name is it? It gets/fetches it using the rules.

Reply:
Yes, you're right. I'll rename this function to `getEventNameFromRules` but happy to take a suggestion if you have one.